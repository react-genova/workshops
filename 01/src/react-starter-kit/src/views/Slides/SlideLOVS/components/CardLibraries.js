//@flow
import React from 'react';
import { Card, CardElement } from '../../../Card';

const CardLibraries = () => (
    <Card title="Libraries" description="and yes, there's no jQuery">
        <CardElement title="axios" description="a fetch library" url="https://github.com/axios/axios"  />
        <CardElement title="lodash" description="the mother of all underscore" url="https://lodash.com/"  />
        <CardElement title="immutable" description="immutable, from heaven to hell and back" url="https://facebook.github.io/immutable-js/"  />
    </Card>
);

export default CardLibraries;