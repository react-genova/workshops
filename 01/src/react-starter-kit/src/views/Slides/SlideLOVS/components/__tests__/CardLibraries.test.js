//@flow
import React from 'react';
import { shallow } from 'enzyme';
import CardLibraries from '../CardLibraries';

describe('Card Libraries', () => {

    it('snapshot check', () => {
        const component = shallow(<CardLibraries />);
        expect(component).toMatchSnapshot();
    });

});