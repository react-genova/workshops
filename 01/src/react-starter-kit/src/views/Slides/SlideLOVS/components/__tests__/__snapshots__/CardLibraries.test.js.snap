// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Libraries snapshot check 1`] = `
<Card
  description="and yes, there's no jQuery"
  title="Libraries"
>
  <CardElement
    description="a fetch library"
    title="axios"
    url="https://github.com/axios/axios"
  />
  <CardElement
    description="the mother of all underscore"
    title="lodash"
    url="https://lodash.com/"
  />
  <CardElement
    description="immutable, from heaven to hell and back"
    title="immutable"
    url="https://facebook.github.io/immutable-js/"
  />
</Card>
`;
